### DISCUSSION TITLE
Double Time Badge Color Based on Multiplier

### DISCUSSION THREAD

**Discussion by mastermach50** [ppy/osu#23250] [Ideas]
It would be nice to color the badges of mods with adjustable values based on the value that was being used.

Like DT at 2x having a darker red than regular DT and DT at1.09x having a lighter shade of red

![osu color adjust](https://user-images.githubusercontent.com/64970593/233294919-421b2fe2-65e4-4859-a521-978f03cbbcc3.png)

This sample was made by adjusting chroma (I am not the best at this)
Some other color value can be changed for a different look, like hue.

**Comment by Rekunan** [comment]
I would much rather prefer a method that tells me the rate without having to compare with anything else, and since there hasn't been anything similar to this for any of the other mods before, and therefore a new feature to be implemented, I'm unsure if dev effort should be allocated for this.
